src/staticdata.c

```diff
@@ -3195,6 +3194,7 @@ static jl_value_t *jl_validate_cache_file(ios_t *f, jl_array_t *depmods, uint64_
 // TODO?: refactor to make it easier to create the "package inspector"
 static jl_value_t *jl_restore_package_image_from_stream(ios_t *f, jl_image_t *image, jl_array_t *depmods, int completeinfo)
 {
+    JL_TIMING(LOAD_IMAGE, LOAD_Pkgimg);
```

Comment:
For a future PR it would be good I guess to also annotate this trace point with the name of the package getting loaded.